js/hang-ui/src/Components/stats/Stats.tsx

```diff
@@ -0,0 +1,36 @@
+import { type Context, Show, useContext } from "solid-js";
+import { StatsWrapper } from "./components/StatsWrapper";
+import { StatsContext } from "./context";
+import styles from "./style/index.scss?inline";
+import type { HandlerProps } from "./types";
+
+interface StatsProps<T = unknown> {
+	context: Context<T>;
+	getElement: (ctx: T) => HandlerProps | undefined;
+}
+
+/**
+ * Stats component for displaying real-time media streaming metrics
+ * Accepts a generic context and a function to extract the media element
+ */
+export const Stats = <T = unknown>(props: StatsProps<T>) => {
+	const contextValue = useContext(props.context);
+
+	return (
+		<div class="stats">
+			<style>{styles}</style>
+			<Show when={props.getElement(contextValue)}>
+				{(element) => (
```

Comment:
What do you think about using an `_` for the name to indicate it's intended to be unused.